src/util/accuracy.js

```diff
@@ -0,0 +1,41 @@
+
+
+ export function calculatePairAccuracy(internPairs) {
+    let isPairedByLocation = JSON.parse(sessionStorage.getItem('isPairedByLocation'));
```

Comment:
I think you may be able to export and import variables rather than using sessionStorage.

Reply:
After doing some quick research I found that using session storage is the best way to go about this to keep the variables most up to date.